### PR TITLE
Changelog for v2.115.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,63 @@
...
+Automatic Migrations
+--------------------
+- The ``inet:flow:dst:softnames`` and ``inet:flow:dst:softnames`` properties
+  are migrated from ``it:dev:str`` to ``it:prod:softname`` types.
+  (`#2930 <https://github.com/vertexproject/synapse/pull/2930>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Features and Enhancements
+-------------------------
+- Updates the  ``inet`` model.
```
Comment (Cisphyx) [inline CHANGELOG.rst:19]: Updates "to" the

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,63 @@
...
+  (`#2930 <https://github.com/vertexproject/synapse/pull/2930>`_)
+
+  ``inet:flow``
+    The ``dst:softnames`` and ``src:softnames`` properties had their types
+    changed from ``it:dev:str`` values to ``it:prod:softname``.
+
+- Add support for for secondary property pivots where the target property
+  is an array type.
+  (`#2928 <https://github.com/vertexproject/synapse/pull/2928>`_)
```
Comment (Cisphyx) [inline CHANGELOG.rst:28]: should be 2922

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,63 @@
...
+Bugfixes
+--------
+- Storm Dmons now default to the using the user default view when the view is
+  not specified in the Dmon definition. Previously the dmon would run in the
+  Cortex default view.
+  (`#2929 <https://github.com/vertexproject/synapse/pull/2929>`_)
+- Non-integer mask values provided to ``inet:cidr4`` types now raise a
+  ``BadTypeValu`` exception.
+  (`#2932 <https://github.com/vertexproject/synapse/pull/2932>`_)
+- Fix a incorrect call to ``os.unlink`` in ``synapse.tools.aha.enroll``.
```
Comment (Cisphyx) [inline CHANGELOG.rst:55]: 'an' incorrect

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,63 @@
...
+  containers that allow a user to hook the container boot process.
+  (`#2919 <https://github.com/vertexproject/synapse/pull/2919>`_)
+- Temporary files created by the Axon, Cortex and base Cell class are now
+  created in the cell local ``tmp`` directory. In many deployments, this would
+  be located in ``/vertex/storage/tmp``.
+  (`#2925 <https://github.com/vertexproject/synapse/pull/2925>`_)
+
+Bugfixes
+--------
+- Storm Dmons now default to the using the user default view when the view is
```
Comment (whippit) [inline CHANGELOG.rst:48]: the using the

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,63 @@
...
+  not specified in the Dmon definition. Previously the dmon would run in the
```
Comment (whippit) [inline CHANGELOG.rst:49]: Dmon vs dmon

File: CHANGELOG.rst (hunk 4 above)
Comment (vEpiphyte) [reply CHANGELOG.rst]: Rewrote this

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**